src/WaveTrack.cpp

```diff
@@ -90,6 +90,9 @@ Track::LinkType ToLinkType(int value)
    return static_cast<Track::LinkType>(value);
 }
 
+constexpr const char* SplitLeftClipPostfix{ " 1" };
+constexpr const char* SplitRightClipPostfix{ " 2" };
```

Comment:
Again I would make a format that substitutes the clip name, if I really do this.

Is this split-clip naming convention good?  I wonder.  Repeated splits might pile up " 1 1 1" ... repeatedly and be undesirable.  But I'm not ruling on that design question.

Reply:
I raised same question in https://github.com/audacity/audacity/issues/1182 there were no objections then

Reply:
When making copies of a copy it's not unusual to get names like "filename - copy - copy - ...", but maybe there is a better approach for clip naming, I don't know.